src/AppConfig.d.ts

```diff
@@ -33,6 +39,8 @@ export interface ServerSettings {
   qidoPathPrefix?: string
   wadoPathPrefix?: string
   stowPathPrefix?: string
+  retryOptions: RetryOptions
+  errorMessages: ErrorMessageSettings[]
```

Comment:
I would make these optional

```suggestion
  retry?: RetryRequestSettings
  errorMessages?: ErrorMessageSettings[]
```